pkg/helpers/tokencmd/basicauth.go

```diff
@@ -113,18 +146,26 @@ var basicRegexes = []*regexp.Regexp{
 }
 
 func basicRealm(headers http.Header) (bool, string) {
+	matchedBootstrap := false
```

Comment:
nit: I think we should now document what `basicRealm` does in a godoc + please write a unit test :)

Reply:
godoc added, unit test cases added